test/ragnarok/ManagedPortfolio.test.ts

```diff
@@ -44,6 +44,7 @@ describe('ManagedPortfolio', () => {
       token.address,
       bulletLoans.address,
       YEAR,
+      parseUSDC(10**7)
```

Comment:
```suggestion
      parseUSDC(1e7)
```
seems to be much more readable